### PR TITLE
Devel version of Jenkins Job Builder (v2.0.0.02b).

### PR DESCRIPTION
**Pull request by alkesh**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

devel version of jenkins job builder (v2.0.0.0b2).
also – fixed errors raised by `brew audit --strict`

### CODE REVIEW

**Comment by ilovezfs** [comment]
Please do not close a PR and open a new copy of it. Instead, squash commits by running `brew update; git checkout $YOUR_BRANCH; git rebase --interactive origin/master` and then mark each commit other than the first as `squash` or `fixup` then `git push -f`. Thanks!

**Comment by alkesh** [comment]
@ilovezfs ah, sorry, I didn't realise what the process was.
Is there anything I need to do now? I may have closed a few pull-requests (#16022, #16018, #16017, #16014) because force-pushing didn't seem to update the pull-request to point at the new commit.

**Comment by ilovezfs** [comment]
Force pushing will refresh the PR status as long as you're pushing to the same branch. In this PR's case that is `alkesh:jenkins-job-builder-devel` (i.e. https://github.com/alkesh/homebrew-core/tree/jenkins-job-builder-devel).

>Is there anything I need to do now?

Yes, you'll need to create a `stable do` block, and then move the dependencies that differ between stable and devel into the `stable do` and `devel do` blocks, respectively. See the `gotools` resources in the `go` formula for an example.

The stable `url` and `sha256` then also move into the `stable do` block.

**Review by fxcoudert — APPROVED** [review]

File: Formula/jenkins-job-builder.rb [hunk 1]
```diff
@@ -13,44 +21,56 @@ class JenkinsJobBuilder < Formula
...
+    url "https://files.pythonhosted.org/packages/08/58/e21f4691e8e75a290bdbfa366f06b9403c653642ef31f879e07f6f9ad7db/stevedore-1.25.0.tar.gz"
+    sha256 "c8a373b90487b7a1b52ebaa3ca5059315bf68d9ebe15b2203c2fa675bd7e1e7e"
+  end
+
   def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
-    %w[pyyaml ordereddict python-jenkins pbr six pip multi_key_dict].each do |r|
+    packages_to_install = %w[pyyaml ordereddict python-jenkins pbr six pip multi_key_dict]
+    packages_to_install << "stevedore" if build.devel?
+    packages_to_install.each do |r|
```
Comment (ilovezfs) [inline Formula/jenkins-job-builder.rb]: should be able to just use `resources` here

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by alkesh** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
Thanks! Merged as 9e4a17249aebb093879fcaa50c4b1c4ae6e76dfb.

**Pull request closed without merging**